modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -114,10 +120,17 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ExceptionBody'
+        '423':
+          description: Cannot acquire lock because notice is already locked.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionBody'
         '500':
           description: |-
             Possible issues:
             - The provided TED API key cannot be currently validated.
+            - Ted Monitor connection issue.
```

Comment:
This is not really understandable for our users (they don't know what TED Monitor is).

Reply:
OK